spec/models/model_spec.rb

```diff
@@ -87,7 +87,7 @@
   end
 
   context "supporting API" do
-    Brassica::Api.readable_models.each do |klass|
+    Api.readable_models.each do |klass|
       context klass do
         it "includes Filterable" do
           expect(klass.ancestors).to include(Filterable)
```

Comment:
One additional request - I think there is value in providing read only access to tables like TaxonomyTerm, that do not support filtering. It kind of works out of the box (I just tested it and both index and show do). Can we exclude them from the requirement of being Filterable (as long as they do not list query/search/fetch in their props/params)?